### ISSUE TITLE
qsub: cannot connect to server (null) (errno=15137) could not connect to trqauthd

### ISSUE THREAD

**Issue by lningyawen** [ComparativeGenomicsToolkit/cactus#533]
Hi,

I've been struggling in a problem about a PBS submission. I have installed the software and tested that it works with `cactus jobStore examples/evolverMammals.txt examples/evolverMammals.hal --root mr` . Then I tested with a subset of my own data in `cactus-prepare-toil` and found a problem:
 `socket_connect_unix failed: 15137
qsub: cannot connect to server (null) (errno=15137) could not connect to trqauthd`. And the submitted script is as follows: 

#!/bin/bash
#PBS -N cactus_alignment
#PBS -l nodes=1:ppn=25
export PATH=/pwd/cactus-bin-v2.0.1/bin:$PATH
export PYTHONPATH=/pwd/cactus-bin-v2.0.1/submodules:$PYTHONPATH
export TOIL_TORQUE_ARGS="-q mem3T"
export TOIL_TORQUE_REQS="walltime=240:00:00"
source /pwd/cactus-bin-v2.0.1/venv/bin/activate
cd /worddirector/
cactus-prepare-toil  ./jobstore my.seqFile \
  --binariesMode local \
  --batchSystem torque \
  --realTimeLogging \
  --disableCaching \
  --outHal /workdirectory/my.hal \
  --defaultDisk 20 \
  --defaultMemory 12 \
  --defaultCores 25 
`
I googled this error message and got the answer that the problem was on the pbs system. Actually, I am not sure if this error is related to my pbs system as I am able to submit and run other programs correctly, especially the example you provided. I've been trying to fix this error for a few days, I've changed the installation method (source package installation or compiled file installation) and the python installation that I was using with conda, among other things, etc. How do you think this problem should be solved? Any help will be much appreciated, Thanks.

Yawen

**Comment by diekhans** [comment]
this is a problem with the interaction of Toil and PBS, please copy this ticket to

   https://github.com/DataBiosphere/toil

and we can engage with them to debug.

**Comment by lningyawen** [comment]
Hi

Thank you for such a quick reply. I have submitted this question to https://github.com/DataBiosphere/toil. Thanks again for your help!

Yawen

**Comment by diekhans** [comment]
thanks for filing the issue with Toil.

**Comment by lningyawen** [comment]
Hi

Because I submitted this question to https://github.com/DataBiosphere/toil, however a few days have passed and I have not received any answer. I would like to ask you, what should I do now? Can you point me in the direction to this error?

> thanks for filing the issue with Toil.

**Comment by lningyawen** [comment]
> thanks for filing the issue with Toil.

Hello

I found out how to solve this problem. I was previously using `qsub cactus.pbs`  to submit commands, however I would get the above error `socket_connect_unix failed: 15137 qsub: cannot connect to server (null) (errno=15137) could not connect to trqauthd`. 

I saved the commands as a `cactus.sh` file and submitted it as `bash cactus.sh` without the above error. 

Actually, I not quite sure how this works, but at least the problem is solved. And thanks for building such a good software.
